### ISSUE TITLE
Delayed overlay loading

### ISSUE THREAD

**Issue by PavanLuca** [languagetool-org/languagetool#9126]
I could replace or correct the typos manually in a glance and LanguageTool overlay appeared anyway.

![Schermata del 2023-07-24 22-23-25](https://github.com/languagetool-org/languagetool/assets/123581658/b221434a-6e33-4cdf-8cb8-63b666a4ea90)

![Schermata del 2023-07-29 21-56-45](https://github.com/languagetool-org/languagetool/assets/123581658/14faffe2-ffe0-43fe-8d24-d8edbf7ddb3e)

![Schermata del 2023-07-31 15-04-02](https://github.com/languagetool-org/languagetool/assets/123581658/a9d3e922-499e-4c6c-a312-c7c48c170912)

![Schermata del 2023-07-31 17-52-49](https://github.com/languagetool-org/languagetool/assets/123581658/6106cc29-55b5-466d-be66-3b9ce81291c3)

**Comment by danielnaber** [comment]
Thanks for the report. I'm not sure I understand, though. Is this the browser add-on or the desktop app? Could you send a short video showing the issue?

**Comment by PavanLuca** [comment]
It's the Firefox extension.
https://drive.google.com/file/d/1ghn_5JrrOTLXRLdV-kza1s8ep4NhAAhl/view?usp=sharing

**Comment by danielnaber** [comment]
Thanks. I cannot reproduce this yet - does it happen everywhere or only on specific websites?

**Comment by danielnaber** [comment]
To make sure I understand (I can't see from your video when you click): you click the underlined word, then edit the word, and after a delay, the overlay opens? Do you have other browser add-ons running? Maybe these slow down your browser? I don't think we had that report before, so I don't think this is caused by just a slow computer.

**Comment by PavanLuca** [comment]
I don't think that extensions slow down my browser, the memory usage is almost the same the whole time.
![Schermata da 2023-08-16 17-53-13](https://github.com/languagetool-org/languagetool/assets/123581658/7029f245-6c95-49fa-8a00-77a31cd99da8)
Do you want to see a certain log or something more specific? From Firefox activities management tab I can create a profile of all threads for specific processes for 5 seconds, from Firefox developers console I can get some logs and then I can share any other info if you tell me where and how to collect them, till now I know about these 2.

**Comment by danielnaber** [comment]
Which browser version do you use and what kind of computer (Windows? CPU, memory)?

**Comment by PavanLuca** [comment]
Firefox 116.0.2 (64 bit) for Ubuntu canonical - 1.0.
![Informazioni sul sistema](https://github.com/languagetool-org/languagetool/assets/123581658/1b0ad623-4c99-49f1-bd57-8379908e12a5)
